core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -262,9 +269,11 @@ private[spark] class CoarseGrainedExecutorBackend(
 
   override def statusUpdate(taskId: Long, state: TaskState, data: ByteBuffer): Unit = {
     val resources = taskResources.getOrElse(taskId, Map.empty[String, ResourceInformation])
-    val msg = StatusUpdate(executorId, taskId, state, data, resources)
+    val cpus = taskCpus.getOrElse(taskId, 0)
```

Comment:
Could we get the taskCpu by `executor.runningTask(taskId).taskDescription.cpus` to get rid of `taskCpus`?

Reply:
Thanks, that looks better. Making the change.

Reply:
done